c2cgeoportal/lib/lingua_extractor.py

```diff
@@ -77,13 +77,13 @@ def __call__(self, filename, options):
         with open(filename) as config_file:
             config = yaml.load(config_file)
             # for application config (.build/config.yaml)
-            if "raster" in config:
+            if "vars" in config:
```

Comment:
yes it's not 100% equivalent but the latest test is a little bit a hack, I prefer the new one :-)